docs/sources/configure/bp-configure.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Configuration best practices
+menuTitle:  Best practices
+description: Grafana Loki configuration best practices
+weight:  100
+---
+# Configuration best practices
+
+Grafana Loki is under active development, and we are constantly working to improve performance. But here are some of the most current best practices for configuration that will give you the best experience with Loki.
```

Comment:
should we qualify "current" with a version?

Reply:
Do we have any idea what version of Loki these best practices were written against?